src/pages/home/report/ParticipantLocalTime.js

```diff
@@ -42,14 +42,15 @@ class ParticipantLocalTime extends PureComponent {
 
     getParticipantLocalTime() {
         const reportRecipientTimezone = lodashGet(this.props.participant, 'timezone', CONST.DEFAULT_TIME_ZONE);
-        moment.locale(this.props.preferredLocale);
-        const reportRecipientDay = moment().tz(reportRecipientTimezone.selected).format('dddd');
-        const currentUserDay = moment().tz(this.props.currentUserTimezone.selected).format('dddd');
+        const timezone1 = DateUtils.getLocalMomentFromTimestamp(this.props.preferredLocale, reportRecipientTimezone);
+        const timezone2 = DateUtils.getLocalMomentFromTimestamp(this.props.preferredLocale, this.props.currentUserTimezone.selected);
```

Comment:
`this.props.currentUserTimezone.selected` This seems to be wrong based on the `DateUtils.getLocalMomentFromTimestamp` implementation. Aren't we supposed to send `this.props.currentUserTimezone` as we are checking selected property inside the function

Reply:
Yeah, I agree with Rajat that this looks wrong. I actually think it makes it a better utility method if it does not check the selected property inside of `getLocalMomentFromTimestamp()`, so I would be in favor of removing `.selected` from inside the method, and then passing `currentUserTimezone.selected` from here.

Reply:
updated!